oauth2/authorization-code-flow/no-token-generation/nginx.conf

```diff
@@ -97,12 +101,14 @@ http {
       proxy_pass https://CHANGE_ME_API_BACKEND/CHANGE_ME_TOKEN_ENDPOINT;
     }
 
+
     location = /threescale_authrep {
       internal;
       set $provider_key CHANGE_ME_PROVIDER_KEY;
 
-      proxy_pass https://threescale_backend/transactions/authrep.xml?provider_key=$provider_key&service_id=$service_id&$usage&$credentials;
+      proxy_pass $threescale_backend/transactions/authrep.xml?provider_key=$provider_key&service_id=$service_id&$usage&$credentials&log%5Bcode%5D=$arg_code&log%5Brequest%5D=$arg_req&log%5Bresponse%5D=$arg_resp;
       proxy_set_header  Host  "su1.3scale.net";
+      proxy_set_header  X-3scale-User-Agent "nginx$deployment";
```

Comment:
The thinking behind this format was that if `$deployment` is empty the header value you get is just `nginx` rather than `nginx/`.
But I agree that it just moves the weirdness out to the environment variable, since you have to set it to something like `/ami` or `/oauth`.